augly/image/transforms.py

```diff
@@ -1310,6 +1310,64 @@ def apply_transform(
             image, factor=self.factor, seed=self.seed, metadata=metadata
         )
 
+    
+class Skew(BaseTransform):
+    def __init__(self, level: float = 0.5, axis: str = "x-axis", p: float = 1.0):
+        """
+        @param level: the level of skew to apply to the image; a larger absolute value will
+            result in a more intense skew. Recommended range is between [-2, 2]
+            
+        @param axis: the axis along which the image will be skewed; can be 'x-axis' or 'y-axis'
+        """
+        super().__init__(p)
+        self.level = level
+        self.axis = axis
+        
+    def apply_transform(
+        self, image: Image.Image, metadata: Optional[List[Dict[str, Any]]] = None
+    ) -> Image.Image:
+        """
+        Skews an image with respect to its x or y-axis
+        
+        @param image: PIL Image to be augmented
+        
+        @param metadata: if set to be a list, metadata about the function execution
+            including its name, the source & dest width, height, etc. will be appended to
+            the inputted list. If set to None, no metadata will be appended or returned
+            
+        @returns: Augmented PIL Image
+        """
+        return F.skew(image, level=self.level, axis=self.axis, metadata=metadata)
+
+
+class Skew(BaseTransform):
+    def __init__(self, level: float = 0.5, axis: str = "x-axis", p: float = 1.0):
+        """
+        @param level: the level of skew to apply to the image; a larger absolute value will
+            result in a more intense skew. Recommended range is between [-2, 2]
+            
+        @param axis: the axis along which the image will be skewed; can be 'x-axis' or 'y-axis'
+        """
+        super().__init__(p)
+        self.level = level
+        self.axis = axis
+        
+    def apply_transform(
+        self, image: Image.Image, metadata: Optional[List[Dict[str, Any]]] = None
+    ) -> Image.Image:
+        """
+        Skews an image with respect to its x or y-axis
+        
+        @param image: PIL Image to be augmented
+        
+        @param metadata: if set to be a list, metadata about the function execution
+            including its name, the source & dest width, height, etc. will be appended to
+            the inputted list. If set to None, no metadata will be appended or returned
+            
+        @returns: Augmented PIL Image
+        """
+        return F.skew(image, level=self.level, axis=self.axis, metadata=metadata)
```

Comment:
it seems like this was duplicated -- can you remove this duplicate class please :D

Reply:
Done !!